packages/prisma-client/src/prisma.ts

```diff
@@ -16,13 +16,37 @@ declare global {
 
 const logPrisma = process.env.NODE_ENV === "production";
 
+const getPgBouncerUrl = () => {
+  // The URL constructor does not support the "postgresql" protocol, so we replace it with "https"
+  // to ensure that the connection is made over HTTPS instead of plain text.
+  const databaseUrl = new URL(
+    process.env.DATABASE_URL!.replace("postgresql://", "https://")
```

Comment:
I would throw instead of silencing types

Reply:
We already check it at env check.
I would not init prisma client here at all. Must be rewritten to init at main app

Reply:
sure, but throwing it makes it clear